tests/kafkatest/services/verifiable_producer.py

```diff
@@ -91,6 +92,10 @@ def _worker(self, idx, node):
                              self.compression_types[compression_index])
             producer_prop_file += "\ncompression.type=%s\n" % self.compression_types[compression_index]
 
+        if self.acks is not None:
+            self.logger.info("VerifiableProducer (index = %d) will use acks = %s", idx, self.acks)
```

Comment:
Not sure if you need to log this here since on line 100, the full set of props gets logged?